lib/Service/FileService.php

```diff
@@ -2,21 +2,44 @@
 
 namespace Stripe\Service;
 
-class FileService extends AbstractService
+class FileService extends \Stripe\Service\AbstractService
```

Comment:
It's handled directly in the `create()` method (since only file creation requests go through `files.stripe.com`): https://github.com/stripe/stripe-php/blob/eb78ac337e221660e31679c8c9f9961cb8645c6a/lib/Service/FileService.php#L53-L56. The method is generated via an override since the implementation is so custom.